tpe/CMakeLists.txt

```
@@ -0,0 +1,2 @@
+add_subdirectory(lib)
+add_subdirectory(plugin)
```

```suggestion
add_subdirectory(plugin)

```
oops, looks like I forgot to add new line